### ISSUE TITLE
USB camera on Raspberry

### ISSUE THREAD

**Issue by svh1985** [brutella/hkcam#25]
Can I also use a 720p USB camera with a Raspberry?

**Comment by libewa** [comment]
I think, no, because it uses the CSI of Pi Zero W

**Comment by brutella** [comment]
Yes, it should work. You have to find out which input device (`/dev/video?`) represents the USB camera and set the launch arguments accordingly.

`--input_filename=/dev/video?`

**Comment by libewa** [comment]
> Yes, it should work. You have to find out which input device (`/dev/video?`) represents the USB camera and set the launch arguments accordingly.

`/dev/video` **IS** the CSI! Probably it's `/dev/usbX`. But I can't test, I don't have any cam, not for CSI, not for USB.

**Comment by brutella** [comment]
> /dev/video IS the CSI! Probably it's /dev/usbX

No, I connected a USB camera and multiple `/dev/video` devices did show up. One of them was for the camera stream.

**Comment by libewa** [comment]
/dev/video IS the CSI! Probably it's /dev/usbX

No, I connected a USB camera and multiple /dev/video devices did show up. One of them was for the camera stream.

OK...
